packages/form-core/test/choice-group/ChoiceGroupMixin.test.js

```diff
@@ -121,6 +136,19 @@ describe('ChoiceGroupMixin', () => {
     expect(el.formElements[2].checked).to.be.true;
   });
 
+  it('can set initial formattedValue on creation', async () => {
```

Comment:
should we also check with `multipleChoice`?

Reply:
added 👍